packages/store-cache/src/utils.ts

```diff
@@ -205,3 +215,10 @@ function serializeKey(key: unknown) {
   if (typeof key === "bigint") return String(key);
   return key;
 }
+
+/**
+ * Helper to concat namespace and table with a separator
+ */
+function toSelector(namespace: string, table: string) {
+  return namespace + "/" + table;
```

Comment:
how does this manifest in our infra as a whole? I've been using `:` as a separator in the string representation of a `TableId` object (currently utils but planning to move to common) and in dev tools displays

Reply:
this is just a local helper to combine updates of a table in `subscribe`, but it never leaves the `subscribe` util since we do `Object.values` in line 152